apps/passport-client/src/validateState.ts

```diff
@@ -175,14 +184,29 @@ export function getRunningAppStateValidationErrors(
     errors.push("missing 'identity'");
   }
 
+  if (!identityV4) {
+    errors.push("missing 'identityV4'");
+  }
```

Comment:
Another check to add: if identityV3 and identityV4 both exist, then they should match via a call to v3tov4Identity

Reply:
I've decided to remove the `identityV4` field from `appState` as it is currently always a pure function of the `identityV3` field. keeping them in sync is complexity that I don't want to deal with right now.